libraries/vault_config.rb

```diff
@@ -37,6 +37,7 @@ class VaultConfig < Chef::Resource
       attribute(:tls_cert_file, kind_of: String)
       attribute(:tls_key_file, kind_of: String)
       attribute(:cache_size, kind_of: Fixnum)
+      attribute(:disable_cache, equal_to: [true, false], default: false)
```

Comment:
When I tested, the config .json didn't contain the option if not explicitly given as chef attribute, but I don't mind removing the default either. The different docs use different best practices, and here poise makes things even more interesting :)